recipe/meta.yaml

```diff
@@ -285,6 +314,68 @@ outputs:
         https://docs.nvidia.com/cuda/cuquantum/license.html
       doc_url: https://docs.nvidia.com/cuda/cuquantum/latest/custatevec/
       dev_url: https://github.com/NVIDIA/cuQuantum
+  
+  - name: cuquantum-python
```

Comment:
Did you figure out why we were unable to get the required behavior from conda-smithy before? By that I mean, having conda-smithy render a conda_build_config with all of the python versions? I would be cautious about relying on this behavior if it is going to be unreliable.

Reply:
Yes. There was a version constraint in the cuquantum-python requirements sections `python >=3.10` that was introduced in the 24.08 release (for which only python312 builds of cuquantum-python exist), but wasn't present prior. I switched back to enforcing the version in the skip statement while leaving python in the requirements unconstrained. I can't judge whether it is safe to rely on this behaviour, but it seems that conda-smithy detects whether a package is a python package for each output separately from the requirements.